turbolinks/src/main/java/com/basecamp/turbolinks/TurbolinksHelper.java

```diff
@@ -167,4 +170,15 @@ private static void configureWebViewDefaults(WebView webView) {
 
         webView.setWebChromeClient(new WebChromeClient());
     }
+
+    /**
+     * Sets the WebView's width/height layout params to MATCH_PARENT
+     *
+     * @param webView The shared webView.
+     */
+    private static void setWebViewLayoutParams(WebView webView) {
+        FrameLayout.LayoutParams params = new FrameLayout.LayoutParams(
+        ViewGroup.LayoutParams.MATCH_PARENT, ViewGroup.LayoutParams.MATCH_PARENT);
```

Comment:
I'd just have this be one long line in line 180 instead of line breaking. We're generally OK with long lines.